matter_server/server/server.py

```diff
@@ -133,6 +133,7 @@ async def start(self) -> None:
             )
         self.loop = asyncio.get_running_loop()
         self.loop.set_exception_handler(_global_loop_exception_handler)
+        self.loop.set_debug(os.environ.get("PYTHONDEBUG") == "1")
```

Comment:
Nit: Definition of this environment variable seems to be debugging is enabled if this string is "non-empty" (https://docs.python.org/3/using/cmdline.html#envvar-PYTHONDEBUG), so 

```suggestion
        self.loop.set_debug(os.environ.get("PYTHONDEBUG") != "")
```

Reply:
well then we also need to check for None

Reply:
Corrected the code to assig a default of empty string